.github/CODEOWNERS

```diff
@@ -1382,7 +1382,94 @@ x-pack/test_serverless/**/test_suites/observability/ai_assistant @elastic/obs-ai
+/x-pack/test/upgrade/ftr_provider_context.d.ts @elastic/appex-qa
+/x-pack/test/upgrade/config.ts @elastic/appex-qa
+/x-pack/test/functional_basic/ftr_provider_context.d.ts @elastic/appex-qa
+/x-pack/test/functional/services/remote_es/remote_es.ts @elastic/appex-qa
+/x-pack/test/functional/services/random.js @elastic/appex-qa
+/x-pack/test/functional/services/index.ts @elastic/appex-qa
+/x-pack/test/functional/page_objects/index.ts @elastic/appex-qa
+/x-pack/test/functional/ftr_provider_context.ts @elastic/appex-qa
+/x-pack/test/functional/README.md @elastic/appex-qa
+/x-pack/test/examples/config.ts @elastic/appex-qa
+/x-pack/test/common/services/index.ts @elastic/appex-qa
+/x-pack/test/common/services/bsearch_secure.ts @elastic/appex-qa
+/x-pack/test/common/ftr_provider_context.ts @elastic/appex-qa
+/x-pack/test/api_integration_basic/ftr_provider_context.d.ts @elastic/appex-qa
+/x-pack/test/api_integration_basic/config.ts @elastic/appex-qa
+/x-pack/test/api_integration_basic/apis/index.ts @elastic/appex-qa
+/x-pack/test/api_integration/services/index.ts @elastic/appex-qa
+/x-pack/test/api_integration/services/es_supertest_without_auth.js @elastic/appex-qa
+/x-pack/test/api_integration/config.ts @elastic/appex-qa
+/x-pack/test/api_integration/apis/kibana/config.ts @elastic/appex-qa
+/x-pack/test/tsconfig.json @elastic/appex-qa
+/x-pack/test/load @elastic/appex-qa
+/test/tsconfig.json @elastic/appex-qa
+/test/plugin_functional/services/index.ts @elastic/appex-qa
+/test/plugin_functional/README.md @elastic/appex-qa
+/test/plugin_functional/config.ts @elastic/appex-qa
+/test/kibana.jsonc @elastic/appex-qa
+/test/harden @elastic/appex-qa
+/test/functional/services/supertest.ts @elastic/appex-qa
+/test/functional/services/remote_es/remote_es.ts @elastic/appex-qa
+/test/functional/services/query_bar.ts @elastic/appex-qa
+/test/functional/services/menu_toggle.ts @elastic/appex-qa
+/test/functional/services/listing_table.ts @elastic/appex-qa
+/test/functional/screenshots/.empty @elastic/appex-qa
+/test/common/fixtures/plugins/coverage @elastic/appex-qa
+/test/functional/services/index.ts @elastic/appex-qa
+/test/functional/services/global_nav.ts @elastic/appex-qa
+/test/functional/services/flyout.ts @elastic/appex-qa
+/test/functional/services/filter_bar.ts @elastic/appex-qa
+/test/functional/services/field_editor.ts @elastic/appex-qa
+/test/functional/services/embedding.ts @elastic/appex-qa
+/test/functional/services/doc_table.ts @elastic/appex-qa
+/test/functional/services/data_grid.ts @elastic/appex-qa
+/test/functional/services/combo_box.ts @elastic/appex-qa
+/test/functional/page_objects/unified_field_list.ts @elastic/appex-qa
+/test/functional/page_objects/time_picker.ts @elastic/appex-qa
+/test/functional/page_objects/index.ts @elastic/appex-qa
+/test/functional/page_objects/header_page.ts @elastic/appex-qa
+/test/functional/page_objects/error_page.ts @elastic/appex-qa
+/test/functional/page_objects/context_page.ts @elastic/appex-qa
```

Comment:
Afaik the tests using this PO are managed by @elastic/kibana-data-discovery , I think it is fair to assign PO as well

Reply:
https://github.com/elastic/kibana/pull/199767/commits/a153e18b4869a53770712afde5c92f275ae3ff8c